### ISSUE TITLE
[Feature] Add dark theme to phpMyAdmin

### ISSUE THREAD

**Issue by maisondasilva** [hestiacp/hestiacp#4128]
### Describe the feature or change in detail

Is it possible to add dark theme for phpMyAdmin in a new version of Hestia?

https://www.phpmyadmin.net/themes/#pma_5_2

darkwolf 5.2
https://files.phpmyadmin.net/themes/darkwolf/5.2/darkwolf-5.2.zip

or BooDark 1.0.1

Thanks

### Would you like to sponsor this feature to have it implemented?

Yes

**Comment by Benzitczo** [comment]
Good dark better.

**Comment by istiak101** [comment]
@maisondasilva I can submit a PR if you can sponsor me through ko-fi.

But accepting the PR is up to the maintainers.

https://ko-fi.com/istiak101

Wouldn’t a small post install script suffice?

**Comment by jaapmarcus** [comment]
> @maisondasilva I can submit a PR if you can sponsor me through ko-fi.
> 
> But accepting the PR is up to the maintainers.
> 
> https://ko-fi.com/istiak101
> 
> Wouldn’t a small post install script suffice?

cd /usr/share/phpmyadmin/themes/
wget https://files.phpmyadmin.net/themes/darkwolf/5.2/darkwolf-5.2.zip
unzip darkwolf-5.2.zip
rm darkwolf-5.2.zip
nano /etc/phpmyadmin/config.inc.php

and add
$cfg['ThemeDefault'] = 'darkwolf'; 

Works fine

**Comment by jaapmarcus** [comment]
Update:

cd /usr/share/phpmyadmin/themes/
wget https://files.phpmyadmin.net/themes/darkwolf/5.2/darkwolf-5.2.zip
unzip darkwolf-5.2.zip
rm darkwolf-5.2.zip

Is enough theme can be changed from inside ...

**Comment by maisondasilva** [comment]
> Update:
> 
> cd /usr/share/phpmyadmin/themes/ wget https://files.phpmyadmin.net/themes/darkwolf/5.2/darkwolf-5.2.zip unzip darkwolf-5.2.zip rm darkwolf-5.2.zip
> 
> Is enough theme can be changed from inside ...

Thanks Share

If I update Hestia doesn't it remove it? That would be my doubt and that's why I haven't sent it yet!

**Comment by jaapmarcus** [comment]
We should not update

It might be done during phpmyadmin update not sure.

Bit it should be easy to include it on default.. Only thing is how it get updated with 5.3 or 6.0 comes out ..

**Comment by maisondasilva** [comment]
> Não devemos atualizar
> 
> Isso pode ser feito durante a atualização do phpmyadmin não tenho certeza.
> 
> Bit deve ser fácil de incluí-lo no padrão.. A única coisa é como ele se atualizar com 5.3 ou 6.0 sai ..

Yes this is true!

Could you update your command list above and add the command to remove the folder if the person doesn't like the theme and wants to download another one?

I will use this Pull as a reference if other people ask

Thanks
Maison

**Comment by jaapmarcus** [comment]
To remove them ...

```
rm -fr /usr/share/phpmyadmin/themes/darkwolf
```

**Comment by maisondasilva** [comment]
Should I close this request, or do you think it might be interesting?

I just tested here, the best theme is the one below, the others have bugs in some words that are not possible to read!

https://files.phpmyadmin.net/themes/boodark/1.1.0/boodark-1.1.0.zip

Regarding the dark mode file manager, I saw which one is used and opening a dark mode request, who knows, maybe he will receive it too!

https://github.com/filegator/filegator/issues/425

Thanks
Maison

**Comment by vipxr** [comment]
I like the dark theme but the problem is the version of phpmyadmin might be updated and the theme become outdated so you have to keep track of it to make sure it still works

**Comment by jaapmarcus** [comment]
> I like the dark theme but the problem is the version of phpmyadmin might be updated and the theme become outdated so you have to keep track of it to make sure it still works

Major issue is you need to track if it is still supported.. PHPmyadmin will get for sure updates ...

**Comment by vipxr** [comment]
> > I like the dark theme but the problem is the version of phpmyadmin might be updated and the theme become outdated so you have to keep track of it to make sure it still works
> 
> Major issue is you need to track if it is still supported.. PHPmyadmin will get for sure updates ...

Yes 100% because they are provided by third parties not the phpMyAdmin team and they must be updated manually when a new release comes

**Comment by AysadKozanoglu** [comment]
only idea: i think it is time to say bye bye to phpMyAdmin and say hello to ADMINER https://www.adminer.org/en/

**Comment by wojsmol** [comment]
@AysadKozanoglu Latest release of Adminer was v 4.8.1, 2021-05-14 so it looks like Adminer is not maintained.

**Comment by Benzitczo** [comment]
> only idea: i think it is time to say bye bye to phpMyAdmin and say hello to ADMINER https://www.adminer.org/en/

so? but is old have 2021 no update version XD

**Comment by AysadKozanoglu** [comment]
> > only idea: i think it is time to say bye bye to phpMyAdmin and say hello to ADMINER https://www.adminer.org/en/
> 
> so? but is old have 2021 no update version XD

well right. but phpMyAdmin is so terrible :(x

**Comment by jaapmarcus** [comment]
There is already a discussion on the use of Adminer. Please keep it on topic ...

**Comment by AysadKozanoglu** [comment]
> @AysadKozanoglu Latest release of Adminer was v 4.8.1, 2021-05-14 so it looks like Adminer is not maintained.

İ would like to maintain th repo, but nobody answers from the owner of repo.

So I will do fork and begin to maintaining the repo.

HestiaMyAdmin would the name of the fork

**Comment by jaapmarcus** [comment]
Don't call it HestiaMyAdmin.

I don't see any reason why it should be called like that it has nothing to do with us...
